docs/pages/modules/get-started.mdx

```diff
@@ -4,19 +4,128 @@ sidebar_title: Get started
 description: Learn about getting started with Expo modules API.
 ---
 
+import { BookOpen02Icon, Grid01Icon } from '@expo/styleguide-icons';
+
 import { BoxLink } from '~/ui/components/BoxLink';
+import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
+If you need to extend the capabilities of your application, you can install expo packages or third party react native libraries.
+Sometimes, this is not enough. You may need to write custom native code to tap into obscure platform APIs or use custom Android or iOS dependencies.
+
+Luckily, Expo has first class support for this in the form of Expo Modules.
+
+Expo Modules allow you to write native code in a way that feels natural with minimal boilerplate that is also consistent on both platforms.
+
+<BoxLink
+  title="Expo Modules Overview"
+  description="Learn more about the design considerations behind Expo Modules."
+  href="./overview"
+  Icon={BookOpen02Icon}
+/>
+<BoxLink
+  title="Expo Modules API Reference"
+  description="A list of building blocks you can use to create your own Expo Module."
+  href="./module-api"
+  Icon={Grid01Icon}
+/>
+
 There are two ways to get started with the Expo Module API: you can either initialize a new module from scratch or add the Expo Module API to an existing module. This guide will walk you through creating a new module from scratch, and the [Integrating in an existing library guide](/modules/existing-library) covers the latter case.
```

Comment:
```suggestion
There are two ways to get started with the Expo Modules API: initializing a new module from scratch or adding the Expo Modules API to an existing library. This guide walks you through creating a new module from scratch. See [Integrating Expo Modules API in an existing library guide](/modules/existing-library) for the second use case.
```

There is an inconsistency between "Expo Module" and "Expo Modules". I think we should stick with one. In the title we say "Expo Modules", so maybe that's the preference.

Reply:
Yeah. That's probably obvious, but my suggestion is to use "Expo Modules API" when we're talking about the API in general and "Expo module" or "Expo modules" when we talk about packages using Expo Modules API (or the legacy one).

Reply:
Thanks for the clarification, @tsapeta. Sometimes, things are obvious to us but might not be to the developer reading the doc (and we don't want them to confuse 😅)